new-dti-website/components/team/MemberGroup.tsx

```diff
@@ -0,0 +1,352 @@
+            <div className="flex flex-col w-3/12">
+              <div className="flex items-center">
+                <h3 className="text-lg font-medium text-[#877B7B]">GRAD</h3>
+                <svg width="100%" height="2" className="ml-3">
+                  <path d="M 0 1 L 500 1" stroke="#E4E4E4" strokeWidth="3px" />
+                </svg>
+              </div>
+              <p>{props.graduation || 'January 1000'}</p>
+            </div>
+            <div className="flex flex-col w-5/12">
+              <div className="flex items-center">
+                <h3 className="text-lg font-medium text-[#877B7B]">MAJOR</h3>
+                <svg width="100%" height="2" className="mx-[3%]">
+                  <path d="M 0 1 L 500 1" stroke="#E4E4E4" strokeWidth="3px" />
+                </svg>
+              </div>
+              <p>{props.major || 'Computer Science'}</p>
+            </div>
+            <div className="w-4/12">
+              <h3 className="text-lg font-medium text-[#877B7B]">HOMETOWN</h3>
+              <p>{props.hometown || 'Mars'}</p>
+            </div>
+          </div>
+        </div>
+        <div className="flex flex-col gap-2">
+          <h2 className="font-semibold text-xl">About</h2>
+          <p>
+            {props.about || 'An amazing member of DTI, but not amazing enough to write an about.'}
+          </p>
+        </div>
+        <div className="flex gap-24">
+          <div className="flex flex-col gap-2">
+            <h3 className="font-semibold text-xl">Subteam</h3>
+            <div
+              className={`flex text-lg ${
+                subteamLink ? 'border-white border-b-black border-2' : ''
+              }`}
+            >
+              {subteamLink ? (
+                <a href={subteamLink} className="whitespace-nowrap">
+                  {subteamName}
+                </a>
+              ) : (
+                <p>{subteamName}</p>
+              )}
+              {subteamLink && <Image src="/icons/link.svg" alt="link" height={20} width={20} />}
+            </div>
+          </div>
+          <div className="flex flex-col gap-2">
+            <h3 className="font-semibold text-xl">Connect</h3>
+            <div className="flex gap-6 items-center">
+              {Object.keys(connectIcons).map((icon) => {
+                const iconDetails = connectIcons[icon];
+                return (
+                  iconDetails.link && (
+                    <a href={iconDetails.link} key={icon}>
+                      <Image
+                        src={iconDetails.src}
+                        alt={icon}
+                        height={iconDetails.height}
+                        width={iconDetails.width}
+                      />
+                    </a>
+                  )
+                );
+              })}
+            </div>
+          </div>
+          <div>
+            <button
+              onMouseEnter={() => setHover(true)}
+              onMouseLeave={() => setHover(false)}
```

Comment:
Ok so I've since learned that using inline arrow functions like this is kinda "frowned upon." It's supposedly less performant (although very marginally) because it needs to be redefined for each re-render but more importantly I think it's a little less readable (although also debatably marginally).  

It's theoretically better to have 
`const mouseHandler = () => setHover(prev => !prev)` and `onMouseEnter={mouseHandler}`

But feel free to ignore, this is just an FYI for the sake of sharing information.